ci/gpu/build.sh

```diff
@@ -46,16 +46,14 @@ nvidia-smi
 logger "Activate conda env..."
 source activate gdf
 conda install "cudatoolkit=$CUDA_REL" \
-              "cupy>=6.5.0" "numpy>=1.16" \
+              "cython>=0.29.14,<3.0.0a0" \
+              "libhwloc" "ucx" "ucx-proc=*=gpu" \
+              "numpy>=1.16" \
+              "pytest" "pytest-asyncio" \
+              "cupy>=6.5.0" \
               "cudf=${MINOR_VERSION}" "dask-cudf=${MINOR_VERSION}" \
               "dask>=2.8.1" "distributed>=2.8.1" \
-              -c rapidsai-nightly
-
-# needed for asynccontextmanager in py36
-conda install -c conda-forge "async_generator" "automake" "libtool" \
-                              "cmake" "automake" "autoconf" "cython>=0.29.14,<3.0.0a0" \
-                              "pytest" "pkg-config" "pytest-asyncio" \
-                              "pynvml" "libhwloc" "psutil"
+              "pynvml" "psutil"
```

Comment:
This still doesn't follow the conventions of the other repos. I'd rather see this instead:

```
conda install "cudatoolkit=$CUDA_REL" \
              "cudf=${MINOR_VERSION}" "dask-cudf=${MINOR_VERSION}" \
              "rapids-build-env=${MINOR_VERSION}"
```

All of your listed packages are in [rapids-build-env](https://github.com/rapidsai/integration/blob/branch-0.15/conda/recipes/rapids-build-env/meta.yaml) with versions shared by all of RAPIDS.

The only version that does not match is `cython` which if necessary can be updated [here](https://github.com/rapidsai/integration/blob/branch-0.15/conda/recipes/versions.yaml#L55)

FWIW `rapids-build-env` is already installed in the gpuCI images so this extra install is only needed to pick up any updates that have happened since they were built overnight.

Reply:
Submitted the `cython` requirement in PR ( https://github.com/rapidsai/integration/pull/93 ).

Though I think we are still in awkward situation with `rapids-build-env` because of `ucx-py` being included. That said, it's interesting to hear `rapids-build-env` is already included. Maybe we should work on dropping `ucx-py` from `rapids-build-env` then?

Independently there's still value in tracking dependencies of `ucx-py` outside of normal RAPIDS processes as we will need to include these when upstreaming to UCX.

Reply:
I'm +1 on https://github.com/rapidsai/integration/pull/94, UCX-Py is not mandatory for most RAPIDS packages.

Reply:
This is done in the latest commit. Thanks for reviewing all of those PRs Mike! 😄